libs/cdk/utils/directives/truncate/truncate.directive.ts

```diff
@@ -83,3 +83,20 @@ export class TruncateDirective implements OnChanges, AfterViewInit {
         this._truncateTarget.style.cssText = this.fdkTruncateState ? this._truncationStyle : this._defaultStyle;
     }
 }
+
+@Directive({
+    selector: '[fdkTruncateTitle], [fdTruncateTitle], [fd-truncate-title]',
+    standalone: true
+})
+export class TruncateTitleDirective implements AfterContentChecked {
```

Comment:
I'm not sure about the directive name. It does not truncate anything, rather adds title attribute to the element if text IS truncated